app/scripts/modules/ecs/src/serverGroup/configure/wizard/CloneServerGroup.ecs.controller.js

```diff
@@ -168,7 +168,6 @@ module(ECS_SERVERGROUP_CONFIGURE_WIZARD_CLONESERVERGROUP_ECS_CONTROLLER, [
     }
 
     function initializeWatches() {
-      $scope.$watch('command.subnetType', createResultProcessor($scope.command.subnetTypeChanged));
```

Comment:
Is there any need of removing this particular code? If we are playing around subnetTypes and not subnetType then leaving this code as is would not harm new changes, right?

Reply:
Correct, no harm in leaving it there. I removed it during my debugging since I never could get it to trigger for the wizard. I will add it again.